src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -467,7 +440,9 @@ update_shell(unsigned int boardIdx, DSAInfo& candidate)
   // Program the shell
   boost::format programFmt("[%s] : %s...\n");
   std::cout << programFmt % flasher.sGetDBDF() % "Updating base (e.g., shell) flash image";
-  update_shell(boardIdx, candidate.file, candidate.file);
+  std::vector<std::string> validated_image;
+  validated_image.push_back(candidate.file);
```

Comment:
Do not obfuscate the code
```suggestion
  std::vector<std::string> validated_image { candidate.file };
```